### PR TITLE
[E2E] Tree View delimiter tests are added

### PR DESCRIPTION
**Pull request by elenaNaboko**

### CODE REVIEW

File: tests/e2e/pageObjects/browser-page.ts [hunk 1]
```diff
@@ -1,692 +1,745 @@
...
+            i++;
+        }
+        return keyValues;
+    }
+
+    /**
+     * Check tree view structure
+     * @folders name of folders for tree view build
+     */
+    async checkTreeViewFoldersStructure(folders: Array<Array<string>>, delimiter: string, commonKeyFolder: boolean): Promise<void> {
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/browser-page.ts:706]: 1. You missed adding a description for the two last parameters.
2. The `Array<Array<string>>` couldn't you use with `string[]`?

File: tests/e2e/tests/critical-path/tree-view/delimiter.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    .meta({type: 'critical_path'})
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneBigConfig, ossStandaloneBigConfig.databaseName);
+    })
+    .afterEach(async() => {
+        await deleteDatabase(ossStandaloneBigConfig.databaseName);
+    })
+test
+    .meta({rte: rte.standalone})
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/tree-view/delimiter.e2e.ts:23]: Could be configured in the fixture level

File: tests/e2e/pageObjects/browser-page.ts (hunk 1 above)
Comment (elenaNaboko) [reply tests/e2e/pageObjects/browser-page.ts]: 1. Added
2. Changed to string[][] (array with arrays)

File: tests/e2e/tests/critical-path/tree-view/delimiter.e2e.ts (hunk 2 above)
Comment (elenaNaboko) [reply tests/e2e/tests/critical-path/tree-view/delimiter.e2e.ts]: Fixed

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**